### PR TITLE
Feat/avatar add title prop

### PR DESCRIPTION
**Pull request by kristof0425**

### CODE REVIEW

File: src/components/avatar/index.js [hunk 1]
```diff
@@ -46,13 +46,13 @@ const Avatar = (props) => {
           name={ props.name === '' ? 'N/A' : props.name }
         />
       ) }
-      { props.caption && (
+      { props.title && (
```
Comment (ruffle1986) [inline src/components/avatar/index.js]: it's not backward compatible but a breaking change. if we release this, we have to change every user avatar in the content editor where we set the name property and set the title property as well.

Comment (ruffle1986) [reply src/components/avatar/index.js]: This is what I wanted to avoid if possible

Comment (ruffle1986) [reply src/components/avatar/index.js]: @kristof0425 ??

**Comment by kristof0425** [comment]
@ruffle1986 I'm closing this PR in favor of #327.

**Pull request closed without merging**